docs/jps_guide/titlepage.tex.in

```diff
@@ -0,0 +1,30 @@
+%% this file is generated automatically. All manual changes will be deleted.
+\newcommand{\HRule}{\rule{\linewidth}{0.5mm}}
+\begin{titlepage}
+\begin{center}
+
+\includegraphics[width=0.75\textwidth]{../images/jupedsim_small.png} \\ [1cm]
+
+\textsc{\LARGE Jülich Pedestrian Simulator} \\ [1.5cm]
+
+\textsc{\small Forschungszentrum Jülich GmbH} \\ [0.5cm]
+
+\HRule \\ [0.4cm]
+{ \huge \bfseries User's Guide Version @PROJECT_VERSION@} \\ [0.4cm]
+
+
+\HRule \\ [1.5cm]
+
+{\large Stand: \today}
```

Comment:
Should this be english too?

Reply:
what exactly do you mean?

Reply:
`Stand` chould be replaced by `State` or something similar as the documentation is in english, right?

Reply:
removed.